cfgov/unprocessed/apps/analytics-gtm/js/youtube-event-listeners-older-style.js

```diff
@@ -18,11 +18,18 @@ import { analyticsLog } from './util/analytics-util';
       'There is already a function defined at window.onYouTubeIframeAPIReady;' +
       ' aborting LunaMetrics Google Analytics YouTube Tracking'
     );
+    return;
+  }
+
+  const iframes = document.getElementsByTagName( 'iframe' );
+
+  if ( iframes.length === 0 ) {
```

Comment:
Could you do `!iframes.length` for brevity and consistency with how we usually check length elsewhere.

Reply:
It would work like that, but since it's talking about a quantity here I'd probably leave it as a number. `!` is casting it to a boolean. Seems kinda obfuscating. https://codepen.io/anon/pen/GdWxVW?editors=1111

Reply:
Correct, but we often do the inverse as `if (array.length) {}` to check existence before doing something as well. Since you're not actually doing anything with the item count, there's nothing wrong with making it a boolean.